README.md

```diff
@@ -68,39 +68,38 @@ For any other type of installation please refer to [Installation doc](https://ww
 ### Bot commands
 
 ```
-usage: main.py [-h] [-v] [--version] [-c PATH] [-d PATH] [-s NAME]
-               [--strategy-path PATH] [--customhyperopt NAME]
-               [--dynamic-whitelist [INT]] [--db-url PATH]
-               {backtesting,edge,hyperopt} ...
+usage: freqtrade [-h] [-v] [--version] [-c PATH] [-d PATH] [-s NAME]
```

Comment:
Honestly, I'm not sure why most documentations say "main.py" ... it won't work as it was (it should be `freqtrade/main.py`) otherwise you're in the wrong folder ...

`freqtrade ...` will work if it's installed correctly using pip... (see `/bin/`) folder... - so i would like to probably slowly) switch the documentation to that...

Reply:
Okay, got it